fun/envs/cms/test.py

```diff
@@ -0,0 +1,84 @@
+from .dev import *
+
+ENVIRONMENT = 'test'
+
+############# Disable useless logging
+import logging
+logging.getLogger("audit").setLevel(logging.WARN)
+logging.getLogger("django_comment_client.utils").setLevel(logging.WARN)
+logging.getLogger('edx.celery.task').setLevel(logging.ERROR)
+logging.getLogger("factory").setLevel(logging.WARN)
+logging.getLogger('instructor_task.api_helper').setLevel(logging.ERROR)
+logging.getLogger('instructor_task.tasks_helper').setLevel(logging.ERROR)
+logging.getLogger("raven.contrib.django.client.DjangoClient").setLevel(logging.WARN)
+logging.getLogger('util.models').setLevel(logging.CRITICAL)
+logging.getLogger('xmodule.modulestore.django').setLevel(logging.ERROR)
+
+########### Imported from edx-platform/lms/envs/test.py
```

Comment:
Je propose de remplacer ce commentaire dans cms/test.py et lms/test.py par un truc du genre : 

    ############ If you modify settings below this line don't forget to modify them both in lms/test.py and cms/test.py